invenio_administration/marshmallow_utils.py

```diff
@@ -36,6 +36,10 @@
     fields.Date: "date",
     fields.TimeDelta: "timedelta",
     fields.Decimal: "decimal",
+    # TODO: This is needed for the is_current_user in the administration
+    # of the user resources. It might be better to implement also a handler for this.
+    # See https://github.com/inveniosoftware/invenio-administration/issues/174
+    fields.Method:  None,
```

Comment:
question: how does jsonschema behaves with `None` or `null` as type? This value will be used below

```python
schema_dict[field].update({
                    "type": custom_mapping[field_type_name],
                })
```

We might need to add a check in the `jsonify_schema` function. Or tests....